components/chunking/chunk-db-write/build.gradle

```
@@ -11,6 +11,7 @@ dependencies {
     implementation platform("net.corda:corda-api:$cordaApiVersion")
     implementation project(":libs:configuration:configuration-core")
     implementation project(':libs:lifecycle:lifecycle')
+    implementation project(':components:virtual-node:virtual-node-rpcops-common')
```

Since `VirtualNodeSender` is passes as a parameter should this not be `api` type dependency? As in:
```suggestion
    api project(':components:virtual-node:virtual-node-rpcops-common')
```